src/components/SearchMetadata.vue

```diff
@@ -11,23 +13,17 @@ export default {
   name: "SearchMetadata",
   props: {
     hits: Number,
-    query: String,
-    status: Object,
+    searchterm: String,
   },
-  computed: {
-    panelHeading: function () {
-      if (this.status.ready == true) {
-        return "Results summary: " + this.hits + " results";
-      } else if (this.status.loading == true) {
-        return "Loading results...";
-      }
-      return "Search Metadata:";
+  methods: {
+    per_page() {
+      return 20;
```

Comment:
We already store this value in Results.vue (line 52 or 55 depending on which branch) - could it be passed into this SFC, rather than re-computed?

Reply:
yeah the whole `per_page` thing is a mess. It needs to be set in a single place for sure but I was nervous having it be ENV because it doesn't "change" what we pull back yet. I think I'll open a ticket to address both moving all of the random places it's set in the app _and_ implementing the TIMDEX call to allow changing it for real.

Reply:
Yeah, if that's better handled in a ticket for focused cleanup, that's fine with me - particularly if the work extends beyond the bounds of this PR (which it seems like it does).

Reply:
https://mitlibraries.atlassian.net/browse/DISCO-209